### ISSUE TITLE
1.12.4 - lua - screens/change_pokemon/change_pokemon.lua:233: attempt to index a nil value - 

### ISSUE THREAD

**Issue by Jerakin** [Jerakin/Pokedex5E#550]
```
1.12.4 - lua - screens/change_pokemon/change_pokemon.lua:233: attempt to index a nil value - 
stack traceback:
	screens/change_pokemon/change_pokemon.lua:233: in function 'redraw_moves'
	screens/change_pokemon/change_pokemon.lua:285: in function 'redraw'
	screens/change_pokemon/change_pokemon.lua:354: in function 'redraw'
	screens/change_pokemon/edit/edit.gui_script:51: in function <screens/change_pokemon/edit/edit.gui_script:42>
	[C]: in function 'resume'
	monarch/monarch.lua:893: in function 'on_message'
	monarch/screen_factory.script:43: in function <monarch/screen_factory.script:35>
```

**Comment by magroader** [comment]
I suspect this is fixed with #564. This callstack looks like something I found and fixed with that PR.

**Comment by Jerakin** [comment]
Highly likely it's cause by it. Closing for now.
